docs/workflows/ios-specific-workflow-steps/build-and-test/xcodebuild-for-unit-and-ui-test.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Xcodebuild for Unit and UI Testing
+description: This step performs unit and UI tests for your iOS applications. This does not "build" your app, but uses the "xcodebuild" command to run tests.
+tags: [build, test, ios, workflow, step]
+sidebar_position: 8
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Xcodebuild for Unit and UI Testing
+
+This step allows you to run unit and UI tests in your project. When this step runs, all your tests are run, and an `.xcresult` file is created as a result. This step does not `build` your app, but uses the `xcodebuild` command to run tests.
+
+:::caution
+
+This step does not generate **IPA**, it only runs tests within the project.
+
+:::
+
+### Prerequisites
+| Prerequisite Workflow Step                      | Description                                     |
+|-------------------------------------------------|-------------------------------------------------|
+| [**Git Clone**](/workflows/common-workflow-steps/build-and-test/git-clone) | The repo needs to be cloned in order to start the running unit and UI test process. After the clone, unit and UI tests will be run. After this step works, the variable `AC_REPOSITORY_DIR` will be created. |
+| [**Xcode Select**](/workflows/ios-specific-workflow-steps/build-and-test/xcode-select)     | This step selects the Xcode version that is specified. |
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE3098-unitOrder.png' />
+
+:::warning
+
+When this step runs, if there is a failed test or tests in the project, it will automatically give an error. This is actually not an error of the Appcircle workflow, but an error caused by the tests in the project finding a failure. In order not to disrupt the workflow flow 
+The option **`"Continue with the next step even if this step fails"`** should be enabled.
```

Comment:
```suggestion
If any tests fail during this step, an error will be automatically reported, which reflects issues within the project's tests, not the Appcircle workflow. To prevent disruption in the workflow, enable the **`"Continue with the next step even if this step fails"`** option.
```